### ISSUE TITLE
`create_user` not working in release 5.0.1

### ISSUE THREAD

**Issue by AbundantSalmon** [workos/workos-python#345]
I am trying to call `create_user` as below

```python
    from workos import WorkOSClient

    workos_client = WorkOSClient(
        client_id=settings.WORKOS_CLIENT_ID,
        api_key=settings.WORKOS_API_KEY,
    )


    test = workos_client.user_management.create_user(
        email="john@example.com",
    )

```

and I am getting a bad exception error:

```
BadRequestException

message=Validation failed, request_id=3dd335f4-a8c7-4548-9d6f-abf4f50e7061, code=invalid_request_parameters, errors=[{'code': 'email_string_required', 'message': 'Email must be a non-empty string.'}])
```

The logging shows that the data is being sent as query parameters:
```
INFO 2024-08-19 07:30:08,028 _client 456 281472661516672 HTTP Request: POST https://api.workos.com/user_management/users?email=john%40example.com "HTTP/1.1 422 Unprocessable Entity"
```

If I call the api directly it works

![Screenshot 2024-08-19 at 4 28 29 PM](https://github.com/user-attachments/assets/2ebcb536-4dbb-4583-b032-0a047438f4cb)

**Comment by PaulAsjes** [comment]
Thanks for the report, we're looking into this and will have a fix up soon.

**Comment by mattgd** [comment]
Hi @AbundantSalmon - this has been fixed in v5.0.2 along with some improved testing surrounding this issue.
